assets/js/components/notifications/OfflineNotification.test.js

```diff
@@ -84,22 +79,15 @@ describe( 'InternalServerError', () => {
 		await waitForRegistry();
 
 		expect(
-			container.querySelector(
-				'.googlesitekit-settings-notice-offline-notice'
-			)
-		).not.toBeNull();
-
-		// eslint-disable-next-line require-await
-		await act( async () => {
-			registry.dispatch( CORE_UI ).setValue( 'isOnline', true );
-		} );
+			queryByText( /you are currently offline/i )
```

Comment:
This is arguably splitting hairs a bit, but we do prefer to use `getByText()` when we do expect the text to exist. Having it throw an error is fine as it will fail the test, which is what we want, and using `getByText()` is a little more specific, we can see from its usage that we definitely expect the text to exist and it will retain its intent if the code is modified.